controllers/roadmap-site/challengeController.js

```diff
@@ -0,0 +1,65 @@
+const logger = require("../../utils/logger");
+const challengeQuery = require("../../models/roadmap-site/challenges");
+const subscribeToChallengeQuery = require("../../models/roadmap-site/subscribe-challenge");
+
+const errorAdminString = "Something went wrong, please try again/contact admin";
+
+const sendChallengeResponse = async (req, res) => {
+  try {
+    if (req.method === "GET") {
+      const allChallenges = await challengeQuery.fetchChallenges();
+      if (allChallenges.length > 0) {
+        return res.status(200).json({
+          message: "Challenges returned successfully!",
+          challenges: allChallenges,
+        });
+      } else {
+        return res.status(404).json({
```

Comment:
Return [boom](https://www.npmjs.com/package/boom) error the way its done in other places.